workshops/migrations/0039_add_permission_groups.py

```diff
@@ -15,7 +15,7 @@ def add_permission_groups(apps, schema_editor):
     # create 'administrators' group with all permissions for CRUD
     auth_ct = ContentType.objects.get_for_models(Permission, Group)
     workshops_ct = ContentType.objects.get_for_models(
-        Airport, Award, Badge, Event, Host, KnowledgeDomain, Lesson, Person,
+        Airport, Award, Badge, Event, Organization, KnowledgeDomain, Lesson, Person,
```

Comment:
This migration has absolute imports:
```py
from workshops.models import (
    Airport, Award, Badge, Event, Organization, KnowledgeDomain, Lesson, Person,
    Qualification, Role, Tag, Task,
)
```
So it would require a change from `Host` to `Organization`.